CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.3.0-beta.1
+- feat: Add support for transaction isolation levels.
+- feat: Adds typed interface for transaction savepoints.
+- feat: Adds support for endpoint definition placement anywhere in server's `src` directory.
+- feat: Adds support for model definitions placement anywhere in server's `src` directory.
+- fix: Improve message transmission guarantee in method streams.
+- chore: Update `vm_service` dependency range.
```

Comment:
We typically don't include chore/refactor in the changelog.

Reply:
Aaah, I found it other changelogs and was uncertain. Will remove it!